### PR TITLE
To do list task

### PR DESCRIPTION
**Pull request by Mikelobi**
### In this project I :

- [ ] Setup webpack to generate output source file for HTML, CSS, Javascript, and images.
- [ ] Added To-do list items dynamically with JavaScript.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        <input type="checkbox"></input>
+        <p>${list.description}</p>
+      </div>
+      <img src="${kebabMenu}" alt="Kebab-menu" class="kebab-icon">
+    `;
+    // let radioBox = document.createElement('input');
+    // radioBox.type = 'checkbox';
+    // listItem.appendChild(radioBox);
+    // listItem.textContent = list.description;
+    // ;
```
Comment (andersonlebon) [inline src/index.js]: - [ ] Kindly remove commented code so that your file will be more clear and clean

**Review by andersonlebon — COMMENTED** [review]
## Hi @Mikelobi 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (andersonlebon) [reply src/index.js:46]: - [ ] Hey @Mikelobi

**Review by aimemalaika — APPROVED** [review]
Hi @Mikelobi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**